assets/dev/js/editor/document/globals/commands/base/disable-enable.js

```diff
@@ -0,0 +1,76 @@
+import CommandHistory from 'elementor-document/commands/base/history';
+import ElementsSettings from 'elementor-document/elements/commands/settings';
+
+export default class DisableEnable extends CommandHistory {
+	static restore( historyItem, isRedo ) {
+		const data = historyItem.get( 'data' );
+
+		// Upon `disable` command toggle `isRedo`.
+		if ( 'document/globals/disable' === data.command ) {
+			isRedo = ! isRedo;
+		}
+
+		historyItem.get( 'containers' ).forEach( ( container ) => {
+			const settings = data.changes[ container.id ],
+				toggle = isRedo ? 'document/globals/enable' : 'document/globals/disable';
+
+			$e.run( toggle, {
+				container,
+				settings,
+			} );
+
+			container.panel.refresh();
```

Comment:
What does this refresh method do?

Reply:
It updates the panel UI in case it was ran from command line.